auto_build/update_level_apps.sh

```diff
@@ -30,6 +30,7 @@ do
 			app_level="${app_level##*: }"
 			app_level=$(echo "$app_level" | cut -d' ' -f1)
 			./change_level.py ${list,,}.json "$app" "$app_level"	# Appel le script change_level.py pour modifier le niveau de l'app dans la liste. ${list,,} permet de passer la variable en minuscule
+			./change_level.py apps.json "$app" "$app_level"	# Appel le script change_level.py pour modifier le niveau de l'app dans la liste.
```

Comment:
Why change_level.py is called two times?

Reply:
The fist one will update the list community or official, the second one is for the list apps.json
During the migration from community/official to apps, we will have the 3 lists. Because we can't just remove a list many people are using.

Reply:
Ok